test/packaging/ansible/roles/package-files-verification/tasks/main.yaml

```diff
@@ -0,0 +1,34 @@
+---
+
+  # this task looks for host distribution/architecture specific from more specific to less specific
+  # Redhat-7.2-aarch64.yaml -> Redhat-7.2.yaml -> Redhat-7-aarch64.yaml -> Redhat-7.yaml
+  # distribution_file_variety is used so i.e. Centos and Redhat can share same configuration
+- name: define package files to verify based on distribution and architecture
+  include_vars: "{{ lookup('first_found', params) }}"
```

Comment:
what's this?

Reply:
It's looping through `files` array below and including the variables defined in the first file found.

This way we can have a list of files based on distribution, version and architecture, i.e. Ubuntu-20, Sles-12.3, Centos-7-aarch64. It will search for a file looking from more specific to less specific. Also covers that Redhat and Centos are using the same file. 

It seemed a way of cover the different scenarios but I might be missing something.